web/releases/nightly.json

```diff
@@ -129,6 +129,28 @@
           ["Hammer_Reference", "Hammer reference"]
         ]
       }
+    },
+    {
+      "title": "Containerized Foreman",
```

Comment:
This is almost certainly not a good name for the new build target and I'm open to suggestions.